rules/windows/defense_evasion_potential_processherpaderping.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/10/27"
+maturity = "production"
+updated_date = "2020/10/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution followed by a file overwrite of an executable by same parent process. This may indicate an
+evasion attempt to execute malicious executable in a stealthy way.
```

Comment:
```suggestion
Identifies the execution followed by a file overwrite of an executable by the same parent process. This may indicate an
evasion attempt to execute malicious code in a stealthy way.
```

Reply:
re-lint

Reply:
no changes after relint